src/ws/frame.rs

```diff
@@ -372,6 +374,10 @@ impl fmt::Display for Frame {
     }
 }
 
+/// A `Binary` representing a `WebSocket` message with framing.
+#[derive(Debug)]
+pub struct FramedBinary(pub Binary);
```

Comment:
Maybe name it `FramedMessage`?

Even `write_raw` says now 😄 

>This is a low-level function that accepts framed messages

Reply:
done